targets/targets.json

```diff
@@ -151,7 +151,7 @@
         "inherits": ["LPCTarget"],
         "core": "Cortex-M0",
         "default_toolchain": "uARM",
-        "extra_labels": ["NXP", "LPC11UXX", "MCU_LPC11U35_501"],
+        "extra_labels": ["NXP", "LPC11UXX", "MCU_LPC11U35_501", "LPC11U35_501"],
```

Comment:
We discourage adding another target's name as an extra label. Further, you cannot do this change because both of the targets in question (`LPC11U35_501` and `XADOW_M0`) are released for mbed 2.

Full explanation: When we do a release of mbed 2, we build all targets that have a `release_versions` key which contains a `2` into a library. After release, when you build for the `XADOW_M0` target, you will pick up two mbed libraries. This is an error. This change thus introduces a bug.